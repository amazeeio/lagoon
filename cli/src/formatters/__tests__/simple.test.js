// @flow

import formatAsSimple from '../simple';

describe('simple formatter', () => {
  it('should format data with 0 data rows correctly', () => {
    const formatted = formatAsSimple([['Name', 'Customer']]);
    expect(formatted).toMatchSnapshot();
  });

  it('should format data with 1 data row correctly', () => {
    const formatted = formatAsSimple([
      ['Name', 'Customer'],
      ['test-project-simple-1', 'test-customer-simple-1'],
    ]);
    expect(formatted).toMatchSnapshot();
  });

  it('should format data with > 1 data rows correctly', () => {
    const formatted = formatAsSimple([
      ['Name', 'Customer'],
      ['test-project-simple-1', 'test-customer-simple-1'],
      ['test-project-simple-2', 'test-customer-simple-2'],
    ]);
    expect(formatted).toMatchSnapshot();
  });
});
