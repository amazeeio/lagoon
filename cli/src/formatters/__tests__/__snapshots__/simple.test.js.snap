// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple formatter should format data with > 1 data rows correctly 1`] = `
"Name: test-project-simple-1
Customer: test-customer-simple-1

Name: test-project-simple-2
Customer: test-customer-simple-2"
`;

exports[`simple formatter should format data with 0 data rows correctly 1`] = `""`;

exports[`simple formatter should format data with 1 data row correctly 1`] = `
"Name: test-project-simple-1
Customer: test-customer-simple-1"
`;
